lib/GADS/Datum/Rag.pm

```diff
@@ -66,6 +68,19 @@ sub write_value
     $self->write_cache('ragval');
 }
 
+sub as_grade
+{
+    my $self = shift;
+    my %mapping = (
```

Comment:
As a general rule I'm removing stuff from the config file, as the new multi-tenanted set up means that more flexibility is needed. It could potentially go in the database, but I think just leave it here for now.